api-specificatie/openapi.yaml

```diff
@@ -769,6 +702,17 @@ components:
       schema:
         type: string
 
+    locatie-query:
```

Comment:
Bij het definieren van querystring parameters kan je aangeven of je explode wil toepassen. default is explode true. In dat geval moet de querystring er dan als volgt uitzien:
```
?locatie=1.0&locatie=2.0
```
Als je explode op false zet, dan moet de querystring er als volgt uitzien:
```
?locatie=1.0,2.0
```

Persoonlijk vind ik explode=false semantisch beter kloppen, maar ik kan mij voorstellen dat het makkelijker is om fouten te maken met de . en ,

Reply:
Dat kunnen we natuurlijk in de description en in het example toelichten. Explode=false past beter bij hoe je coördinaten doorgeeft. Ik was hier al wel tegen aan gelopen, maar wist niet van de explode. Thanks.